package.json

```diff
@@ -26,10 +26,11 @@
     "postinstall": "lerna bootstrap && npm run -s update-greenkeeper-json",
     "prerelease": "npm run build:full && npm run mocha && npm run lint",
     "release": "lerna version && lerna publish from-git --yes",
+    "update-package-lock-json": "lerna exec \"npm install -s --ignore-scripts --package-lock-only --no-audit\"",
```

Comment:
Can we add `git add packages/*/package-lock.json"` at the end of the script as pointed out in https://github.com/lerna/lerna/issues/1415#issuecomment-455790478?

Reply:
Good catch. I forgot that.

Reply:
Fixed.